### DISCUSSION TITLE
Add support for Browser Extensions

### DISCUSSION THREAD

**Discussion by kuloud** [rainbow-me/rainbowkit#2225] [Q&A] [answered]
Due to MV3's restrictions on dynamic import of add-on, integrating rainbowkit into add-on can cause loading failures for locale/svg resources:
```
const src = async () => (await import('./login.svg')).default;
(await import('./en_US.json')).default;
// etc.
```
![image](https://github.com/user-attachments/assets/b2dbd108-85e5-4ec5-8a7e-a44daf50c1ac)

Whether the browser add-on is compatible

**Comment by DanielSinclair** [comment, marked as answer]
Typically you would use RainbowKit on the web, rather than in a wallet extension, as the wallet already has access to the signer — whereas RainbowKit is designed to connect wallets to isolated web dapps. But the import issues you are describing should be able to be overcome by tweaking the bundler to bundle dynamic imports.
